### ISSUE TITLE
TypeError: Cannot read property 'startsWith' of undefined

### ISSUE THREAD

**Issue by iVesha** [HunterLarco/twitter-v2#36]
Here is my code 
```js
let twitter = require("twitter-v2");

let client = new twitter({
    consumer_key: "...",
    consumer_secret: "...",
    access_token_key: "...",
    access_token_secret: "..."
})


const { data } = client.get('search/tweets', { q: 'Kawaki', count: 5 });
console.log(data);
```

and when i run it, i get following error
```txt
C:\Users\<USER>\node_modules\twitter-v2\src\Credent
ials.js:129
    this._bearer_token = bearer_token.startsWith('Bearer ')
                                      ^

TypeError: Cannot read property 'startsWith' of undefined
    at new Credentials (C:\Users\<USER>\node_module
s\←[4mtwitter-v2←[24m\src\Credentials.js:129:39)
    at new Twitter (C:\Users\<USER>\node_modules\←[
4mtwitter-v2←[24m\src\twitter.js:30:14)
    at Object.<anonymous> (C:\Users\<USER>\index.js
:3:14)
←[90m    at Module._compile (internal/modules/cjs/loader.js:1118:30)←[39m
←[90m    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1138:1
0)←[39m
←[90m    at Module.load (internal/modules/cjs/loader.js:982:32)←[39m
←[90m    at Function.Module._load (internal/modules/cjs/loader.js:875:14)←[39m
←[90m    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_ma
in.js:71:12)←[39m
←[90m    at internal/main/run_main_module.js:17:47←[39m
```
what should i do ?

**Comment by zachguo** [comment]
Bug with line I guess https://github.com/HunterLarco/twitter-v2/blob/9dd0ce662a508f521b0d0d02e640d8edd65b4cda/src/Credentials.js#L129

`bearer_token` may not always be provided.

**Comment by HunterLarco** [comment]
This is now released under [1.0.2](https://www.npmjs.com/package/twitter-v2/v/1.0.2)
